larq_compute_engine/tflite/kernels/bconv2d.cc

```diff
@@ -210,11 +216,6 @@ TfLiteStatus Prepare(TfLiteContext* context, TfLiteNode* node) {
     TF_LITE_ENSURE_EQ(context, thresholds->type, kTfLiteInt32);
     TF_LITE_ENSURE_EQ(context, SizeOfDimension(thresholds, 0),
                       bconv2d_params->channels_out);
-    TF_LITE_ENSURE_MSG(context,
-                       bconv2d_params->padding_type != kTfLitePaddingSame ||
-                           bconv2d_params->pad_value == 1,
-                       "Writing bitpacked output is only supported with "
-                       "valid or one-padding.");
```

Comment:
I think we still need this check

Reply:
Oh wait never mind, I'm wrong about that! Yep, we don't need it.

Reply:
Isn't this caught by the general zero-padding check? That one will fail when writing bitpacked output.